package.json

```diff
@@ -18,6 +18,7 @@
 		"file-saver": "^2.0.5",
 		"i18next": "^22.4.14",
 		"i18next-browser-languagedetector": "^7.0.1",
+		"ical.js": "^1.5.0",
```

Comment:
[ical.js](https://www.npmjs.com/package/ical.js) is developed by Mozilla, so I expect it's a pretty solid library.

Reply:
Thanks for selecting a library with a reliable maintainer! ❤️